mlrun/datastore/storeytargets.py

```diff
@@ -43,7 +43,21 @@ def get_url_and_storage_options(path, external_storage_options=None):
 
 class TDEngineStoreyTarget(storey.TDEngineTarget):
     def __init__(self, *args, **kwargs):
```

Comment:
It looks like you can safely change the signature to the more explicit form:
```suggestion
    def __init__(self, *args, *, url: str, **kwargs) -> None:
```